lib/OpenQA/WebAPI/Controller/File.pm

```diff
@@ -76,10 +76,17 @@ sub test_file {
     return $self->serve_static_($self->param('filename'));
 }
 
+sub download_asset {
+    my ($self) = @_;
+    # we handle this in apache
+    return $self->reply->not_found;
```

Comment:
I know, that apache is required right now, but I always keep in mind that this requirement should be "fixed". Given this, should I consider apache/nginx/... as mandatory for the future and stop worrying about it?